GOVERNANCE.md

```diff
@@ -11,21 +11,46 @@ The Fluentd community adheres to the following principles:
 
 ## Voting
 
-The Fluentd project employs "organization voting" to ensure no single organization can dominate the project.
+The Fluentd project employs "Organization Voting" and "Vetoes" to ensure no single organization can dominate the project.
+
+For formal votes, we follow these steps.
+
+1. Have enough period to discuss the topic so that all discussion members agree that there are no more points to be discussed.
+2. Add a specific statement of what is being voted on to the relevant GitHub issue or PR.
+3. Declare the start of voting.
+4. Ask maintainers to indicate their yes/no vote on the issue or PR.
+5. Tally the votes and note the outcome after a suitable period.
```

Comment:
Who does this?

Reply:
> Most of these texts follow the current ones.

> https://github.com/fluent/fluentd/blob/dc7e4b295270d6a521f43331321628dba2692ca4/GOVERNANCE.md#L22

This too.

```
after a suitable period of time, the votes will be tallied and the outcome noted.
```

I think the subject is the person who proposes "Voting".
Currently, the subject is not specified, and I don't see a problem with that.

Do you want to specify it on this occasion?

Reply:
In my idea, this kind of document should define those things explicitly. We can easily imagine the situation:

1. a maintainer (A) started a voting
2. there are minimum "yes" 2 weeks later, but A is really busy
3. A is still busy and does not respond to the voting issue

What will happen in this situation? Another maintainer can close this voting or not?
My idea is, we should make rules as explicit as possible if we have them.

Reply:
I see. Thanks for the explanation.
I understand that we can improve the document from that perspective,
but I think it should be made separately from this PR, as well as https://github.com/fluent/fluentd/pull/4160#discussion_r1206271345.

In this PR, I want to solve #4151.
Other than the threshold of the conditions required to approve the proposal, I keep the essential contents as unchanged as possible. 

I think the problem with that situation comes from the current document, not from this revision.
We should discuss it separately from this revision.